packages/salesforcedx-utils-vscode/test/unit/commands/channelService.test.ts

```diff
@@ -64,17 +69,16 @@ describe('Channel Service', () => {
         .build(),
       {}
     ).execute();
-    // @ts-ignore
     channelService.streamCommandOutput(execution);
 
-    await new Promise<string>((resolve, reject) => {
+    await new Promise<string | void>(resolve => {
       execution.processExitSubject.subscribe(data => {
```

Comment:
since line 70 was changed, might as well change date in the next line to ()

Reply:
done